react/features/authentication/components/native/LoginDialog.tsx

```diff
@@ -140,7 +140,6 @@ class LoginDialog extends Component<IProps, IState> {
 
         return (
             <Dialog.Container
-                coverScreen = { false }
```

Comment:
Why?

Reply:
This prop is no longer available in the latest react-native-dialog. Will add it back with the revert.